### ISSUE TITLE
Support Adwords Events

### ISSUE THREAD

**Issue by harshmaur** [react-ga/react-ga#270]
We were integrating Adwords with our account and we found that, adwords have different parameters in the event. 

```
  gtag('event', 'conversion', {
      'send_to': 'AW-800630262/zKjzCN7qhIQBEPbL4v0C'
  });
```

Is it possible to support the same in `ReactGA.event()` ?

**Comment by DveMac** [comment]
I think the solution proposed here could be adapted to cover this too.

https://github.com/react-ga/react-ga/issues/265

**Comment by Thrajnor** [comment]
can i hope for any example how to make it work? i need to make conversion after user submits my form, but i have no idea how.

**Comment by Deboracgs** [comment]
any example to event conversion ?

**Comment by marisveide** [comment]
Yes, please, here too - need to send the  `gtag('event', 'conversion', {...});`.
How to do it with ReactGA?

**Comment by jffin** [comment]
is somebody find a way to implement google ads with reactGA?
and there are different formats reactGA uses UA-xxx-x format but google ads gives only AW-xxxxxx format? how it can be used all together?

**Comment by keevcodes** [comment]
Seems the trick is to setup an event with ReactGA.event() and creating a goal with the exact details of the event. Then you can import the goal into GoogleAds from GoogleAnalytics. Seems to be working for me. 

https://blacklineit.com/2015/11/24/adwords-conversion-tracking-chat-plugins-spas-ajax-pages/

**Comment by Fluorz** [comment]
+1

**Comment by pueyo5** [comment]
Any update on this?

**Comment by srsandy** [comment]
+1

**Comment by leogoesger** [comment]
+1

**Comment by w90** [comment]
+1

**Comment by miguelcaro** [comment]
+1

**Comment by iamsimonyu** [comment]
Hi all -- I found [this guide](https://dev.to/keevcodes/syncing-up-google-ads-and-analytics-to-your-react-app-516o), seems promising.

Basically, rather than using Google Ads events, the approach is to set a goal in Google Analytics (which can be triggered via ReactGA.event), and import that Google Analytics goal into Google Ads.

**Comment by jffin** [comment]
> Hi all -- I found [this guide](https://dev.to/keevcodes/syncing-up-google-ads-and-analytics-to-your-react-app-516o), seems promising.
> 
> Basically, rather than using Google Ads events, the approach is to set a goal in Google Analytics (which can be triggered via ReactGA.event), and import that Google Analytics goal into Google Ads.

Unfortunately, the link does not work.

**Comment by iamsimonyu** [comment]
hmm link works for me...

**Comment by jffin** [comment]
> hmm link works for me...

Yes, seems to be an issue with my browser.

**Comment by thisismydesign** [comment]
I couldn't get `ReactGA -> Google Ads` to work directly, however, you can connect Google Analytics and Google Ads so that you only need to track to Google Analytics as described here: https://stackoverflow.com/a/63753012/2771889

- Link Google Analytics and Google Ads: https://support.google.com/analytics/answer/1033961
- Set goal in Google Analytics: https://support.google.com/analytics/answer/1032415?hl=en
- Import goal in Google Ads: https://support.google.com/analytics/answer/1034306?hl=en

Both articles above ([1](https://blacklineit.com/2015/11/24/adwords-conversion-tracking-chat-plugins-spas-ajax-pages/), [2](https://dev.to/keevcodes/syncing-up-google-ads-and-analytics-to-your-react-app-516o)) would've described this but they're no longer available.

**Comment by sladg** [comment]
Any updates on this?
Cannot figure out a way how to reasonably track events to Ad Words.

**Comment by Luisquii** [comment]
Any updates on this?
Still not able to use conversions.

**Comment by ayrosochang** [comment]
Any updates?

**Comment by sladg** [comment]
It seems like no one is maintaining this repo anymore :/
